config/exercise_readme.go.tmpl

```diff
@@ -23,7 +23,6 @@ raise Exception("Meaningful message indicating the source of the error")
 
 To run the tests, run the appropriate command below ([why they are different](https://github.com/pytest-dev/pytest/issues/1629#issue-161422224)):
 
-- Python 2.7: `py.test {{ .Spec.SnakeCaseName }}_test.py`
 - Python 3.4+: `pytest {{ .Spec.SnakeCaseName }}_test.py`
 
 Alternatively, you can tell Python to run the pytest module (allowing the same command to be used regardless of Python version):
```

Comment:
Within the same mindset as the above comment, I don't think we need the part in parenthesis here:

> Alternatively, you can tell Python to run the pytest module: